client/galaxy/scripts/mvc/workflow/workflow.js

```diff
@@ -0,0 +1,201 @@
+/** Workflow view */
+define( [ 'mvc/form/form-view',
+          'mvc/ui/ui-misc',
+          'mvc/ui/ui-select',
+          'mvc/tool/tool-form-composite' ], function( Form, Ui, Select, ToolForm ) {
```

Comment:
Ahh, k, I see.  That last merge with dev wasn't necessary, but I can wait on the build and re-merge after if you want.